docs/user-guide/api-mediation/api-mediation-modulith.md

```diff
@@ -0,0 +1,141 @@
+# API Mediation Layer Single-Service deployment
+
+Zowe version 3.3.0 introduces as a technical preview the option to switch execution mode from the current modularized scheme to an All-In-One option.
+This switch <!--"Execution of xyz though the Monulith method ..."-->brings performance benefits and configuration simplification for new installations:
+
+* Unified memory footprint. <!--Please add the benefit of this. -->
+* Minimize JVM processes. 
+* Minimize networking operations.
+* Unified configuration options.
+
+## Architecture
```

Comment:
Please specify the role(s) that performs this task (System Programmer, Network Admin, Security Admin)

Reply:
The selection and most of the configuration will be System Programmer, but some of the changes impact Network Admin.